base/applications/mspaint/winproc.cpp

```diff
@@ -730,6 +730,7 @@ LRESULT CMainWindow::OnCommand(UINT nMsg, WPARAM wParam, LPARAM lParam, BOOL& bH
             break;
         case IDM_VIEWSTATUSBAR:
             ::ShowWindow(hStatusBar, ::IsWindowVisible(hStatusBar) ? SW_HIDE : SW_SHOW);
+            registrySettings.ShowStatusBar = ::IsWindowVisible(hStatusBar);
```

Comment:
You may exchange those two lines, and use in the `ShowWindow` call the resulting value of `registrySettings.ShowStatusBar` ? :thinking:

Reply:
Fixed.